GitCommands/ArgumentBuilder.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections;
+using System.Text;
+using JetBrains.Annotations;
+
+namespace GitCommands
+{
+    /// <summary>
+    /// Builds a command line argument string from zero or more arguments.
+    /// </summary>
+    /// <remarks>
+    /// To retrieve the constructed argument list string, call <see cref="ToString"/>.
+    /// <para />
```

Comment:
:memo: `<para>` should wrap the text, and the line between paragraphs should be empty, like this:

```
/// <remarks>
/// <para>To retrieve the constructed argument list string, call <see cref="ToString"/>.</para>
///
/// <para>Arguments are separated by a single space character.</para>
///
/// <para>Adding <see langword="null"/> or white-space strings has no effect on the output, which can be
/// useful in some calling constructions.</para>
///
/// ...
/// </remarks>
```

Reply:
Again this is more semantically correct but I do find it harder to read, and it does seem to be less common in the wild, at least on codebases I've worked on. It does seem more common in GE though. Will change.

Reply:
Changed and pushed.